src/resources/Validator.js

```diff
@@ -141,6 +141,12 @@ Validator.prototype.validate = function(submission, next) {
   var error = null;
   debug('Starting validation');
 
+  // Skip validation if no data is provided.
+  if (!submission.data) {
```

Comment:
Not really... The PUT method works like a PATCH anywhere where you can exclude the data and it will leave it untouched. Also, if you submit without data it will just have it as an empty submission.